.github/workflows/tests.yml

```diff
@@ -267,7 +267,7 @@ jobs:
 
     - name: Install system dependencies
       run: |
-        sudo apt-get install -y libsnappy-dev libkrb5-dev krb5-user
+        sudo apt-get update -y libsnappy-dev libkrb5-dev krb5-user
```

Comment:
sudo apt-get update && sudo apt-get install -y libsnappy-dev libkrb5-dev krb5-user

Reply:
still failiing ```Err:1 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libgssrpc4 amd64 1.16-2ubuntu0.1
  404  Not Found [IP: 52.147.219.192 80]
Err:2 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libkdb5-9 amd64 1.16-2ubuntu0.1
  404  Not Found [IP: 52.147.219.192 80]
Err:3 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libkadm5srv-mit11 amd64 1.16-2ubuntu0.1
  404  Not Found [IP: 52.147.219.192 80]
Err:4 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libkadm5clnt-mit11 amd64 1.16-2ubuntu0.1
  404  Not Found [IP: 52.147.219.192 80]
Err:6 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 krb5-multidev amd64 1.16-2ubuntu0.1
  404  Not Found [IP: 52.147.219.192 80]
Err:7 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libkrb5-dev amd64 1.16-2ubuntu0.1
  404  Not Found [IP: 52.147.219.192 80]```

Reply:
Try rebasing on master, there was a (very similar) fix there, and it builds successfully.